corehq/form_processor/models.py

```diff
@@ -258,9 +259,7 @@ class AbstractAttachment(models.Model):
 
     @property
     def filepath(self):
-        if getattr(settings, 'IS_TRAVIS', False):
-            return os.path.join('/home/travis/', str(self.attachment_id))
-        return os.path.join('/tmp/', str(self.attachment_id))
+        return os.path.join(gettempdir(), str(self.attachment_id))
```

Comment:
assuming it was a problem because of this: https://github.com/dimagi/commcare-hq/commit/cb002dd7002305c36a84e81eaa6f3976b94df1c2